llvm/lib/Target/AMDGPU/AMDGPULowerExecSync.cpp

```diff
@@ -0,0 +1,241 @@
+//===-- AMDGPULowerExecSync.cpp -----------------------------------------===//
```

Comment:
Remove the file name. It adds no new information, and can become stale if the file is renamed.